doc/old_and_new_proposal.md

```diff
@@ -768,12 +768,8 @@ calculated. If YaST is not able to make space for all the volumes required by
 those default settings, it will perform new attempts altering the settings. For
 that, it will follow the `disable_order` for each volume with that field.
 
-> FIXME: unclear (and not yet implemented anyway)
->
-> `adjust_by_ram` is optional == `adjust_by_ram_configurable`?
-
 In the first iteration, it will look for the lowest number there. If
-`adjust_by_ram` is optional in that volume and enabled, it will disable it. If
+`adjust_by_ram_configurable` is true in that volume, it will disable `adjust_by_ram`. If
```

Comment:
NP: well, it will disable it only in case `adjust_by_ram` is true.

Reply:
Is there a difference, technically?

Reply:
Not, really